src/ui/server/graphql/queries/application.ts

```diff
@@ -46,9 +46,15 @@ const applicationQuery = gql`
           name
           financialUk {
             id
+            accountNumber
```

Comment:
I thought these already existed for playback?

Reply:
Yes, but not in the "All sections" task list context.

The other GQL queries that have this are for showing encrypted data.

For "All sections", we need to check the data exists, but it doesn't have to be decrypted. So we can simply update the standard "Get application" GQL query to return the encrypted) data 👍